packages/dapp/src/components/WelcomeWrapper/WelcomeWrapper.tsx

```diff
@@ -26,6 +27,18 @@ export const WelcomeWrapper: React.FC<
   const [openBridgesPopper, setOpenBridgesPopper] = useState(false);
   const [openDexsPopper, setOpenDexsPopper] = useState(false);
 
+  const auditsWelcomeUrl = appendUTMParametersToLink(
+    'https://docs.li.fi/smart-contracts/audits',
+    {
+      utm_campaign: 'jumper_to_docs',
+      utm_medium: 'landing_page',
+    },
+  );
+  const lifiWelcomeUrl = appendUTMParametersToLink('https://li.fi/', {
+    utm_campaign: 'jumper_to_lifi',
+    utm_medium: 'landing_page',
```

Comment:
consistent naming with welcome screen/page
medium: welcomepage_lifi_btn

Reply:
There is just one link which links to lifi. So I question if it is necessary to append "lifi_btn".